src/server/physics/marbles.js

```diff
@@ -1,10 +1,39 @@
-module.exports = function(Ammo, world) {
+module.exports = function(Ammo, world, mapBuilder) {
+	let _marblesTransformAux = new Ammo.btTransform(),
+		_pos = undefined,
+		_rot = undefined,
```

Comment:
Still need to update the conventions for it, but best use `null` instead of `undefined` here, so it's clear it's a used variable.
Edit: We actually still use it in several places, so instead of going over all of them I'd say fix when you see them is fine too?